Catappult/appcoins-sdk
# [feature/APT-3629] Indicative analytics

**What does this PR do?**

   Adds Indicative and Kotlin to the SDK.
   It currently only adds events for the initial wallet checks and the instalation flow

**Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) **

- Kotlin
- Indicative lib
- Indicative key

**Where should the reviewer start?**

- [ ] SdkAnalytics
- [ ] IndicativeEventLogger

**How should this be manually tested?**

- Run the this version of the SDK with a launcher app (toolbox, Diceroll)
- Check initial instalation events

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APT-3629


**Code Review Checklist**
